book/content/rdm/storagebackup.md

```diff
@@ -7,21 +7,45 @@ suitable storage solutions and backing your data up frequently.
+To add an additional layer of security, you should encrypt devices and/or
 files where needed.
 
-Your institution might provide local storage solutions and policies or guidelines restricting what you can use. Thus, we
-recommend you familiarise yourself with your local policies and recommendations.
+Your institution might provide local storage solutions and policies or guidelines restricting what you can use.
+Thus, we recommend you familiarise yourself with your local policies and recommendations.
 
-When you are ready to release the data to the wider community, you can also search for the appropriate
-[databases and repositories](https://fairsharing.org/databases) in FAIRsharing, according to your data type, and type of
-access to the data. Major
-[publishers are progressively defining clearer recommendations for data deposition and sharing](https://fairsharing.org/recommendations),
-endorsing specific generics as well as domain-specific databases and repositories.
+When you are ready to release the data to the wider community, you can also search for the appropriate [databases and repositories](https://fairsharing.org/databases) in FAIRsharing, according to your data type, and type of access to the data.
+Major [publishers are progressively defining clearer recommendations for data deposition and sharing](https://fairsharing.org/recommendations), endorsing specific generics as well as domain-specific databases and repositories.
+
+### Data organisation
+
+To organise your data you can create a folder structure (or re-use a [folder structure](http://nikola.me/folder_structure.html) to ensure that you are able to find your files.
+
+-	Make sure you have enough (sub)folders so that files can be stored in the right folder and are not scattered in folders where they don’t belong or stored in large quantities in a single folder
+-	Use a clear folder structure: you can structure folders based on the person that has generated the data/folder, chronologically (month, year, sessions), per project (as done in the example above), or based on analysis method/equipment/type of data.
+
+You can also pull/download folder structures using GitHub.
+[This template](https://github.com/bvreede/good-enough-projec) by Barbara Vreede, based on [cookiecutter](https://github.com/cookiecutter/cookiecutter), follows recommended practices for scientific computing by [Wilson et al. (2017)](https://doi.org/10.1371/journal.pcbi.1005510).
+
+### File Naming conventions
+
+Structure your file names and set up a template for this.
+ It is very useful to start with the date (when the file was generated: YYYYMMDD) which will sort your files chronologically and also creates a unique identifier for each file.
+ This process will be immediately clear if there are multiple files generated on the same day that will have to be given a version number –or “A, B”-, because otherwise overwriting would occurs if you store these files in the proper folder).
```

Comment:
```suggestion
 This process will be immediately clear if there are multiple files generated on the same day that will have to be given a version number –or “A, B”-, because otherwise overwriting would occur, if you store these files in the proper folder).
```